### PR TITLE
Persistence

### PR DESCRIPTION
**Pull request by j97b**

### CODE REVIEW

File: src/main/java/com/bae/persistence/domain/Boulder.java [hunk 1]
```diff
@@ -1,5 +1,194 @@
...
+  		this.status = status;
+  		this.attemptDate = attemptDate;
+  		this.completionDate = completionDate;
+  	}
+
+	public long getId() {
+		return id;
+	}
+
+	public void setId(long id) {
```
Comment (LukeCharles555) [inline src/main/java/com/bae/persistence/domain/Boulder.java]: change long to Long

**Comment by j97b** [comment]
changed datasource url to something more sensible

**Comment by j97b** [comment]
I fixed a minor thing

File: src/main/java/com/bae/persistence/domain/Boulder.java [hunk 2]
```diff
@@ -1,5 +1,194 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "BOULDER")
```
Comment (jkiesslinger) [inline src/main/java/com/bae/persistence/domain/Boulder.java]: lower case for sql

File: src/main/java/com/bae/persistence/domain/Boulder.java [hunk 3]
```diff
@@ -1,5 +1,194 @@
...
+  	
+  	public Boulder(String name, String location, String grade, String status, Date attemptDate) {
+  		this.name = name;
+  		this.location = location;
+  		this.grade = grade;
+  		this.status = status;
+  		this.attemptDate = attemptDate;
+  	}
+  	
+ 	public Boulder(String name, String location, String grade, String status, Date attemptDate, Date completionDate) {
```
Comment (jkiesslinger) [inline src/main/java/com/bae/persistence/domain/Boulder.java]: consider a builder for these

File: src/main/java/com/bae/persistence/domain/User.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "USER")
```
Comment (jkiesslinger) [inline src/main/java/com/bae/persistence/domain/User.java]: conventionally lower case

File: src/main/java/com/bae/persistence/domain/User.java [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+public class User {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private long id;
+	@OneToMany(mappedBy="BOULDER")
```
Comment (jkiesslinger) [inline src/main/java/com/bae/persistence/domain/User.java]: mappedBy="id" 
There's no element called BOULDER in Boulder

**Comment by LukeCharles555** [comment]
ok to merge to dev branch

**Pull request merged**